### DISCUSSION TITLE
kb/cryptography/lll

### DISCUSSION THREAD

**Discussion by giscus** [jiegec/kb#25] [General]
# kb/cryptography/lll

Knowledge Base of @jiegec

https://jia.je/kb/cryptography/lll.html

**Comment by n-WN** [comment]
您就是 jiegec

**Comment by tsingkong** [comment]
杰哥的笔记留言可以用 google 账号登陆，但是知识库需要用 github 登陆。
同一个站，两套账号，能不能打通呢

**Reply by jiegec** [reply]
历史遗留问题，数据不好迁移
